tools/packaging/static-build/virtiofsd/build-static-virtiofsd.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2022 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+ARCH=$(uname -m)
+
+script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+source "${script_dir}/../../scripts/lib.sh"
+
+virtiofsd_version="${virtiofsd_version:-}"
+kata_version="${kata_version:-}"
+
+[ -n "$virtiofsd_version" ] || virtiofsd_version=$(get_from_kata_deps "externals.virtiofsd.version" "${kata_version}")
```

Comment:
From the `get_from_kata_deps`:

https://github.com/kata-containers/kata-containers/blob/97425a7fe64cba267649d7791eaef4efa0cffe03/tools/packaging/scripts/lib.sh#L35-L43

It seems the second parameter is not required, and also the `kata_version` can be deleted.

Reply:
@liubin, updated!